puppet/zulip/files/redis/redis.conf

```diff
@@ -385,6 +385,7 @@ slave-priority 100
 # output buffers (but this is not needed if the policy is 'noeviction').
 #
 # maxmemory <bytes>
+maxmemory 512mb
```

Comment:
Hmm, it seems to me like this could create a problem where as the redis cache fills up, rate-limiting data could be kicked out of the redis instance's memory unexpectedly by a bunch of traffic that would previously have gone to memcached.  Is there a way that we can compartmentalize to keep (small) rate-limiting data and similar things that we don't want to get evicted safe from eviction?